docs/provision.md

```diff
@@ -134,6 +134,23 @@ the following config parameters. By default, the tunnel won't be created.
 If you have doubts about how to list the users, please check the following
 [link](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/iap_tunnel_instance_iam#argument-reference).
 
+All instaces create by default a boot disk with the same name as the instance,
+to make it persistent you have to set persistent_disks to true.
+
+- `persistent_disks = true`.
+
+You can also create an extra persistent disk that you can attach to any
+VMs by default it will attach to the same name VM. If disk count is not set or
+the value is `0` it will use the node count value.
```

Comment:
```suggestion
You can also create an extra persistent disk that can be attached to any VMs.
It will be attached to the same name of VM, by default. When the `<node>_disk_count`
is not set or the value is `0`, it will use the `<node>_count` value.
```